lib/environments/meteorEnvironment.js

```diff
@@ -1,22 +1,46 @@
+import fs from 'fs';
+import path from 'path';
+
+import findPackageDependencies from '../findPackageDependencies';
+
+const coreModules = [
+  'meteor/accounts-base',
+  'meteor/blaze',
+  'meteor/check',
+  'meteor/ddp-client',
+  'meteor/ddp-rate-limiter',
+  'meteor/ejson',
+  'meteor/email',
+  'meteor/http',
+  'meteor/check',
+  'meteor/meteor',
+  'meteor/mongo',
+  'meteor/random',
+  'meteor/reactive-var',
+  'meteor/session',
+  'meteor/templating',
+  'meteor/tracker',
+];
+
+function meteorPackageDependencies(
+  { config }: Object
+): Array<string> {
+  const meteorPackagesPath = path.join(config.workingDirectory, '.meteor/packages');
+  if (!fs.existsSync(meteorPackagesPath)) {
+    return [];
+  }
+  const packages = fs.readFileSync(meteorPackagesPath, 'utf8')
+    // extract an array of package names from the packages file
+    .match(/^\s*([^# @\n]+)/gm)
```

Comment:
The `m` flag is definitely working as expected in the code and is needed. The [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags) also includes it.

<package>@<version> is the syntax used in packages to pin a package to a version. I had no need for this particular version.

The best documentation I could find (probably because I trust code more than docs and didn't look extensively at the docs) is the [validatePackageName function](https://github.com/meteor/meteor/blob/devel/packages/package-version-parser/package-version-parser.js#L379) in Meteor's code. It looks for bad characters with the following statement.
```js
  var badChar = packageName.match(/[^a-z0-9:.\-]/);
```
So, "@" isn't allowed.